src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -627,25 +627,44 @@ public void testCanAnnotateSpanningDeletions() {
             for (final Allele allele : alleleToFuncotationMap.keySet()) {
-                if (allele.equals(Allele.SPAN_DEL)) {
-                    final List<String> txIds = alleleToFuncotationMap.get(allele).getTranscriptList();
-                    for (String txId : txIds) {
-                        if (allele.equals(Allele.SPAN_DEL)) {
-                            Assert.assertEquals(alleleToFuncotationMap.get(allele).get(txId).get(0).getField("Gencode_28_variantClassification"), GencodeFuncotation.VariantClassification.COULD_NOT_DETERMINE.toString());
-                        } else {
-                            Assert.assertNotEquals(alleleToFuncotationMap.get(allele).get(txId).get(0).getField("Gencode_28_variantClassification"), GencodeFuncotation.VariantClassification.COULD_NOT_DETERMINE.toString());
-                        }
+                final List<String> txIds = alleleToFuncotationMap.get(allele).getTranscriptList();
+                for (final String txId : txIds) {
+                    if (allele.equals(Allele.SPAN_DEL)) {
+                        Assert.assertEquals(alleleToFuncotationMap.get(allele).get(txId).get(0).getField("Gencode_28_variantClassification"), GencodeFuncotation.VariantClassification.COULD_NOT_DETERMINE.toString());
+                        Assert.assertEquals(alleleToFuncotationMap.get(allele).get(txId).get(0).getField("Gencode_28_variantType"), GencodeFuncotation.VariantType.NA.toString());
+                    } else {
+                        Assert.assertNotEquals(alleleToFuncotationMap.get(allele).get(txId).get(0).getField("Gencode_28_variantClassification"), GencodeFuncotation.VariantClassification.COULD_NOT_DETERMINE.toString());
                     }
-
                 }
             }
         }
     }
+
+    @Test
+    public void testNoSpanningDeletionWriteWithMAF() {
+        final FuncotatorArgumentDefinitions.OutputFormatType outputFormatType = FuncotatorArgumentDefinitions.OutputFormatType.MAF;
+        final File outputFile = getOutputFile(outputFormatType);
+
+        final ArgumentsBuilder arguments = new ArgumentsBuilder();
+
+        arguments.addVCF(new File(SPANNING_DEL_VCF));
+        arguments.addOutput(outputFile);
+        arguments.addReference(new File(hg38Chr3Ref));
+        arguments.addArgument(FuncotatorArgumentDefinitions.DATA_SOURCES_PATH_LONG_NAME, DS_PIK3CA_DIR);
+        arguments.addArgument(FuncotatorArgumentDefinitions.REFERENCE_VERSION_LONG_NAME, FuncotatorTestConstants.REFERENCE_VERSION_HG38);
+        arguments.addArgument(FuncotatorArgumentDefinitions.OUTPUT_FORMAT_LONG_NAME, outputFormatType.toString());
+
+        runCommandLine(arguments);
+
+        // There should only be one variant in the MAF, not two
+        final AnnotatedIntervalCollection annotatedIntervalCollection = AnnotatedIntervalCollection.create(outputFile.toPath(), null);
```

Comment:
In the other test I think you check to see that the variants are all spanning deletions.  Can you do that here as well just to make sure?

Reply:
I cannot, since MAFs should drop spanning deletions, always.

@jonn-smith

Reply:
Can I merge now?  @jonn-smith

Reply:
Yeah - if it passes go for it.